MaxText/configs/base.yml

```diff
@@ -121,6 +121,15 @@ record_internal_nn_metrics: 0
 # Create a GCS bucket, e.g. my-maxtext-outputs and set this to "gs://my-maxtext-outputs/"
 base_output_directory: ""
 
+# Whether or not to enable emergency checkpoint. If True, `enable_emergency_checkpoint` must also be specified.
```

Comment:
I think what is meant is If True, "local_checkpoint_directory" must also be specified

Reply:
Correct, also added more explanation of the feature.